### PR TITLE
Fix application start duration log "Failed to load class org.slf4j.impl.StaticLoggerBinder."

### PR DESCRIPTION
**Pull request by zifeihan**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [x] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.
Fix application start duration log "Failed to load class "org.slf4j.impl.StaticLoggerBinder"." when use kafka reporter. because kafka reporter use slf4j-api, but no implementor for slf4j.
- How to fix?
kafka client will log some message by slf4j-api,but no org.slf4j.impl.StaticLoggerBinder implementor for slf4j-api, add org.slf4j.impl.StaticLoggerBinder implementor , so we can rewrite kafka log to skywalking log.
___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/src/main/java/org/apache/skywalking/apm/agent/dependencies/org/slf4j/impl/Slf4jLoggerAdapter.java [hunk 1]
```diff
@@ -0,0 +1,342 @@
...
+    }
+
+    @Override
+    public void info(final String format, final Object... arguments) {
+        log.info(format, arguments);
+    }
+
+    @Override
+    public void info(final String msg, final Throwable t) {
+        log.error(t, msg);
```
Comment (dmsolr) [inline apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/src/main/java/org/apache/skywalking/apm/agent/dependencies/org/slf4j/impl/Slf4jLoggerAdapter.java:190]: should be log.info(t, msg); ?

File: apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/src/main/java/org/apache/skywalking/apm/agent/dependencies/org/slf4j/impl/Slf4jLoggerAdapter.java [hunk 2]
```diff
@@ -0,0 +1,342 @@
...
+    }
+
+    @Override
+    public void info(final Marker marker, final String format, final Object... arguments) {
+        log.info(format, arguments);
+    }
+
+    @Override
+    public void info(final Marker marker, final String msg, final Throwable t) {
+        log.error(t, msg);
```
Comment (dmsolr) [inline apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/src/main/java/org/apache/skywalking/apm/agent/dependencies/org/slf4j/impl/Slf4jLoggerAdapter.java:220]: log.info(...) ?

File: apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/src/main/java/org/apache/skywalking/apm/agent/dependencies/org/slf4j/impl/Slf4jLoggerAdapter.java [hunk 3]
```diff
@@ -0,0 +1,342 @@
...
+
+    @Override
+    public void error(final String format, final Object arg1, final Object arg2) {
+
+    }
+
+    @Override
+    public void error(final String format, final Object... arguments) {
+
+    }
```
Comment (dmsolr) [inline apm-sniffer/optional-reporter-plugins/kafka-reporter-plugin/src/main/java/org/apache/skywalking/apm/agent/dependencies/org/slf4j/impl/Slf4jLoggerAdapter.java:306]: still need to delegate ILog

**Review by dmsolr — DISMISSED** [review]

File: apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/logging/core/LogLevel.java [hunk 4]
```diff
@@ -19,5 +19,5 @@
 package org.apache.skywalking.apm.agent.core.logging.core;
 
 public enum LogLevel {
-    DEBUG, INFO, WARN, ERROR, OFF
+    TRACE, DEBUG, INFO, WARN, ERROR, OFF
```
Comment (wu-sheng) [inline apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/logging/core/LogLevel.java:22]: Please help with updating `logging.level` in the agent setup doc. Add all available values in the documentation.

**Review by wu-sheng — COMMENTED** [review]
One question, does grpc-java use `Sl4j`? I remember there is this FAQ, https://github.com/apache/skywalking/blob/master/docs/en/FAQ/Too-many-gRPC-logs.md.

Could you make it to the SkyWalking too? Also, I think this implementation should be a part of agent-core, locating in `agent.core.logging.adaptor`.

File: apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/dependencies/org/slf4j/impl/StaticLoggerBinder.java [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.apm.dependencies.org.slf4j.impl;
```
Comment (wu-sheng) [inline apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/dependencies/org/slf4j/impl/StaticLoggerBinder.java:19]: Why this class needs to locate in `org.apache.skywalking.apm.dependencies.org.slf4j.impl`? I think I have asked you to place this in the `org.apache.skywalking.apm.agent.core.logging.adaptor`

File: apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/dependencies/org/slf4j/impl/StaticLoggerBinder.java [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+import org.slf4j.ILoggerFactory;
+import org.slf4j.spi.LoggerFactoryBinder;
+
+/**
+ * SLF4J LoggerFactoryBinder implementation using Log4j. This class is part of the required classes used to specify an
+ * SLF4J logger provider implementation.
```
Comment (wu-sheng) [inline apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/dependencies/org/slf4j/impl/StaticLoggerBinder.java:26]:
```suggestion
 * The slf4j-api would try to load org.slf4j.impl.StaticLoggerBinder internal. In the agent core, we add our own implementation
 * for bridging to SkyWalking internal log component. 
 * Therefore, logs of netty/grpc/kafka(agent shaded components) would output through the SkyWalking's log.
```

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**